### PR TITLE
COM-85: Refactor Theming of Picker Components

### PR DESCRIPTION
**Pull request by jamesricky**

### CODE REVIEW

File: packages/admin/admin-color-picker/src/PreviewIndicatorColorProps.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// For reasons I cannot explain, this type must be in it's own file, otherwise the import of `ColorPickerColorPreviewProps` in the story is missing this part.
```
Comment (johnnyomair) [inline packages/admin/admin-color-picker/src/PreviewIndicatorColorProps.ts:1]: Probably a cyclic dependency?

**Review by johnnyomair — COMMENTED** [review]
It would be better to have smaller PRs for separate components.

**Comment by jamesricky** [comment]
> It would be better to have smaller PRs for separate components.

Sorry, yes I agree. In this case it was a bit difficult, because they all used `InputWithPopper` as a base. 
Refactoring the base required refactoring the others for everything to work without lint-errors.

File: packages/admin/admin-color-picker/src/PreviewIndicatorColorProps.ts (hunk 1 above)
Comment (johnnyomair) [reply packages/admin/admin-color-picker/src/PreviewIndicatorColorProps.ts]: @jamesricky I tried that locally and "fixed" it by moving the type: d9f1020fec5305678089398fa7f23ac3309751bc

Could you please verify that this works for you as well?

Comment (jamesricky) [reply packages/admin/admin-color-picker/src/PreviewIndicatorColorProps.ts]: Yes this fixes the issue, thanks 🙂

**Review by johnnyomair — APPROVED** [review]

**Review by thomasdax98 — APPROVED** [review]

**Pull request merged**